oauth_tokens/base.py

```diff
@@ -81,7 +81,7 @@ def add_data_credentials(self, data):
         pass
 
     def get_form_data_from_content(self, content, **kwargs):
-        bs = BeautifulSoup(content)
+        bs = BeautifulSoup(content, 'html5lib')
```

Comment:
@ppeczek Are you sure html5lib should be used for every provider? I think it's necessary to test